sdks/python/setup.py

```diff
@@ -142,11 +142,11 @@ def get_version():
     'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
     # [BEAM-4543] Datastore IO is not supported in Python 3.
     'googledatastore>=7.0.1,<7.1; python_version < "3.0"',
-    'google-cloud-pubsub==0.39.0',
+    'google-cloud-pubsub>=0.39.0',
```

Comment:
Adding a upper bound like `google-cloud-pubsub>=0.39.0, <0.40.0` would prevent unexpected breaking changes in the dependency.